app/views/pages/home.html.erb

```diff
@@ -1,18 +1,37 @@
-<div class="banner" id="banner">
 
+<!-- code a "Welcome" section -->
+<div class="banner, text-justify" id="banner">
+  <h1 class="text-center">Welcome to MyStorage</h1>
+<h2 class="text-center">Might be interesting?</h2>
 </div>
 
-<h1>Welcome to MyStorage</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
-
-<h2>Here are some exciting flats </h2>
-<div class="cards">
-  <ul class="list-inline">
-    <li class="list-inline-item">Storage space 1</li>
-    <li class="list-inline-item">Storage space 2</li>
-    <li class="list-inline-item">Storage space 3</li>
-</ul>
-<a href="..." class="btn btn-primary">Explore?</a>
-</div>
-<div>some extra info/features</div>
+
+<!-- code a "Cards" section -->
```

Comment:
Since we might want to be reusing code, it would make sense to have this in a partial as well (DRY + consistency)